integration/form-test.ts

```diff
@@ -783,6 +849,40 @@ test.describe("Forms", () => {
     });
   });
 
+  test.describe("with submitter button having `formMethod` attribute", () => {
+    test("submits with GET instead of POST", async ({ page }) => {
+      let app = new PlaywrightFixture(appFixture, page);
+      await app.goto("/submitter-formmethod");
+      await app.clickElement("text=Submit with GET");
+      await page.waitForLoadState("load");
+      expect(await app.getHtml("pre")).toBe("<pre>GET</pre>");
+    });
+
+    test("submits with POST instead of GET", async ({ page }) => {
+      let app = new PlaywrightFixture(appFixture, page);
+      await app.goto("/submitter-formmethod");
+      await app.clickElement("text=Submit with POST");
+      await page.waitForLoadState("load");
+      expect(await app.getHtml("pre")).toBe("<pre>POST</pre>");
+    });
```

Comment:
Let's either create 1 test that checks both or wrap this with a `'overrides the form "method" attribute with the button "formmethod" attribute'` `describe`.

Reply:
I've wrapped them in a `describe`, as suggested